src/Ubiquity/db/providers/pdo/drivers/PgsqlDriverMetas.php

```diff
@@ -13,19 +13,86 @@
 class PgsqlDriverMetas extends AbstractDriverMetaDatas {
 
 	public function getForeignKeys($tableName, $pkName, $dbName = null): array {
-		// TODO
+		$recordset = $this->dbInstance->query ( '
+			SELECT k1.constraint_catalog as "CONSTRAINT_CATALOG",
+			k1.constraint_schema as "CONSTRAINT_SCHEMA",
+			k1.constraint_name as "CONSTRAINT_NAME",
+			k1.table_catalog  as "TABLE_CATALOG",
+			k1.table_schema  as "TABLE_SCHEMA",
```

Comment:
Alternating tabs and spaces :
Ubiquity only uses tabs to indent

Reply:
ok. i changed.